package-lock.json

```diff
@@ -1,6476 +1,8 @@
 {
   "name": "moodlebooster",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Please don't touch this file.

Reply:
When a contributor do npm install it changes automatically package.lock.
usually there is git.ignore file will solve this problem, I will open issue for this